### PR TITLE
Multi buffer patch

### PR DESCRIPTION
**Pull request by myarjunar**
### What does it fix ?
* Ticket #3893 
* Funded by : DFAT
* Description : add keywords to the output, ordered distance input

### CODE REVIEW

File: safe/gis/vector/multi_buffering.py [hunk 1]
```diff
@@ -122,8 +123,9 @@ def multi_buffering(layer, radii, callback=None):
             callback(current=i, maximum=feature_count, step=processing_step)
 
     # We transfer keywords to the output.
-    buffered.keywords = layer.keywords
+    buffered.keywords = KeywordIO().read_keywords(layer)
```
Comment (myarjunar) [inline safe/gis/vector/multi_buffering.py]: I found that 'layer.keywords' give nothing, it didn't return any keywords. So I use keyword io. Is it fine?

Comment (Gustry) [inline safe/gis/vector/multi_buffering.py]: Hum no, we don't use KeywordIO inside the gis package as the layer should have monkey patching. We assume we have monkey patching everytime. Please keep the old line.

But I'm wondering why you care about keywords from the source layer. The layer shouldn't have keywords as the purpose of the multi-buffer tool is to create the hazard layer, so AFAIK the layer will never have keywords before this step.

File: safe/gui/tools/help/multi_buffer_help.py [hunk 2]
```diff
@@ -75,9 +75,17 @@ def content():
...
+        'complete, check the launch keywords wizard check box. If you want '
+        'to do it later, uncheck the launch keywords wizard check box.'
+    ))
+    tips.add(tr(
         'A new layer will be added to QGIS after the buffering is complete. '
         'The layer will contain new buffer polygon(s) and the class name will '
-        'be stored as an attribute of each polygon.'
+        'be stored as an attribute of each polygon. And if you check the '
+        'launch keywords wizard check box, the keywords creation wizard will '
+        'launch right after the buffering is complete. You can assign the '
```
Comment (Gustry) [inline safe/gui/tools/help/multi_buffer_help.py]: `is completed` ?

File: safe/gis/vector/multi_buffering.py (hunk 1 above)
Comment (myarjunar) [inline safe/gis/vector/multi_buffering.py]: I thought you want the keywords wizard will automatically give the default option on the dialog, so I think I could get it from the source's keywords. Or maybe it's just my misunderstanding :joy: so, stick with the old one?

Comment (Gustry) [inline safe/gis/vector/multi_buffering.py]: Yes we want the keyword wizard to open automatically after your tool.
But this layer won't have keywords at the beginning, right ?

Hum, this algorithm will create a memory layer right ? Then you give this layer to the datastore, right ? Then keywords will be written automatically. Let me check your code again

Comment (Gustry) [inline safe/gis/vector/multi_buffering.py]: But why do you want to **read** keywords ?

File: safe/gui/tools/help/multi_buffer_help.py [hunk 3]
```diff
@@ -75,9 +75,17 @@ def content():
         'remove, then click the remove (-) button.'
     ))
     tips.add(tr(
+        'To launch keywords creation wizard right after the buffering is '
+        'complete, check the launch keywords wizard check box. If you want '
+        'to do it later, uncheck the launch keywords wizard check box.'
```
Comment (timlinux) [inline safe/gui/tools/help/multi_buffer_help.py]: 'Check the "launch keywords wizard" checkbox to launch the 
'keywords creation wizard after the buffering is '
'complete. If you want assign keywords later, uncheck '
'the "launch keywords wizard" checkbox.'

File: safe/gui/tools/help/multi_buffer_help.py [hunk 4]
```diff
@@ -75,9 +75,17 @@ def content():
...
+    ))
+    tips.add(tr(
         'A new layer will be added to QGIS after the buffering is complete. '
         'The layer will contain new buffer polygon(s) and the class name will '
-        'be stored as an attribute of each polygon.'
+        'be stored as an attribute of each polygon. And if you check the '
```
Comment (timlinux) [inline safe/gui/tools/help/multi_buffer_help.py]: And if -> If

File: safe/gui/tools/help/multi_buffer_help.py (hunk 2 above)
Comment (timlinux) [inline safe/gui/tools/help/multi_buffer_help.py]: s/right//g
buffering is complete -> buffering process has completed.

File: safe/gis/vector/multi_buffering.py (hunk 1 above)
Comment (myarjunar) [inline safe/gis/vector/multi_buffering.py]: so no need to change anything for this part?

Comment (myarjunar) [inline safe/gis/vector/multi_buffering.py]: remove the KeywordIO, done

**Pull request merged**